README.md

```diff
@@ -1,189 +1,95 @@
-# SignalFx Smart Agent 
+# SignalFx Smart Agent Quick Install
```

Comment:
The README on an open source project is usually geared towards developers, not users looking for a quick guide.  Make the quick install a separate document from the README (you can move some stuff out of the README if it is too repetitive).

Reply:
As I understood the request from Joe D., he specifically asked that the readme become a quick install doc. Can you guys follow up with him to agree on the plan for Anna Lisa's work? Thanks. (I don't know his github user name so can't tag him here.)

Reply:
Restored the original README and linked to the information from the Quick Install page. The plan is to make the Quick Install the first topic and delete the Overview page. Leave the more technical info in the README when Ben says developers will look for it. Rewrote the Quick Start to lay out the steps.